Test.C

```diff
@@ -165,6 +165,8 @@ int main(){
 
         //spend credits interactive menu
         printf("|-------------------------------- Credit Store --------------------------------|\n");
+        printf("Here you can buy equipment and upgrade encrytption as well as sell the equipment that you already own.");
```

Comment:
```suggestion
        printf("Here you can buy equipment and upgrade encryption as well as sell the equipment/n");
        printf("that you already own.\n");
```
This change allows us to stay within 80 characters throughout the entire game